interface/themes/tabs_style_compact.css

```diff
@@ -29,11 +29,12 @@ html
 }
 body
```

Comment:
btw,
I just realized I needed to empty my web browser cache before testing this. I'll do that and retest this.
-brady